### DISCUSSION TITLE
DataTypes Issue

### DISCUSSION THREAD

**Discussion by Robinz250** [sequelize/sequelize#13558] [Help & Questions]
So.... this is just me being a beginner in Sequelize and not having the firm grasp of it, yet. But on any of my Models the DataTypes object is undefined so I can't use the Datatypes to define my models properties. But when I log the DataTypes variable it clearly shows that the object exists and indeed has all the properties I needed. But It throws me a "Cannot read property ´STRING' of undefined". Can please someone help?

**Comment by jpmmadeira** [comment]
Hello @Robinz250, nice to see you here 🙂
Have you tried to use Sequelize import, instead of DataTypes?

For example:
`name: Sequelize.STRING` instead of `name: DataTypes.STRING`
